osquery/tables/system/windows/users.cpp

```diff
@@ -220,7 +220,7 @@ void processLocalAccounts(const std::set<std::string>& selectedUids,
         // Will return empty string on fail
         auto sid = LPUSER_INFO_4(userLvl4Buff)->usri4_user_sid;
         auto uid = getUidFromSid(sid);
-        auto gid = LPUSER_INFO_4(userLvl4Buff)->usri4_primary_group_id;
+        auto gid = getGidFromSid(sid);
```

Comment:
And yes, unfortunately I had to undo the speedier way to retrieve `GID` per user, but I don't think this is major.